houston/queries.go

```diff
@@ -19,15 +19,15 @@ var (
 		$label: String!
 		$type: String = "airflow"
 		$version: String = "0.7.5"
-		$workspaceUuid: Uuid!
+		$workspaceId: Id!
 	) {
 		createDeployment(
 			label: $label
 			type: $type
 			version: $version
-			workspaceUuid: $workspaceUuid
+			workspaceId: $workspaceId
```

Comment:
@andriisoldatenko I think we may have to keep the parameter names, as they are sent to houston, with the old UUID naming. Internally, we should use the `id` terminology, as well is in any labels that show up while using the CLI. 

I would think that this would fail due to `workspaceId` being sent on this line, instead of `workspaceUuid`, as  seen here: https://github.com/astronomer/houston-api/blob/master/src/schema.graphql#L290.

I think it should work if you just change this line (and similar lines in other mutations to `workspaceUuid: $workspaceId`, keeping it internally named `workspaceId`, just sending `workspaceUuid` to Houston. We can patch up Orbit/Houston/CLI to fully switch to `...Id` format next, all at once.